pkgs/development/python-modules/dm-tree/default.nix

```diff
@@ -24,7 +36,10 @@ buildPythonPackage rec {
     hash = "sha256-VvSJTuEYjIz/4TTibSLkbg65YmcYqHImTHOomeorMJc=";
   };
 
-  patches = [ ./cmake.patch ];
+  patches = [
+    patchCMakeAbseil
+    patchCMakePybind
+  ] ++ (lib.optional stdenv.isDarwin ./0003-don-t-configure-apple.patch);
```

Comment:
why get rid of `cmake.patch`? also why can't we download `0003-don-t-configure-apple.patch`?

Reply:
It's my patch, figured it out locally. 

As for cmake patch I replaced the combined patch with two smaller ones that copy the logic from conda forge, it appeared to be a good source of inspiration for "how to debug this"

Reply:
gotcha, makes sense